sonar-jmeter-plugin/pom.xml

```diff
@@ -18,7 +18,7 @@
   <properties>
     <sonar.pluginClass>org.sonar.plugins.jmeter.JMeterPlugin</sonar.pluginClass>
     <sonar.pluginName>JMeter</sonar.pluginName>
-    <sonar.version>3.7</sonar.version>
+    <sonar.version>5.1</sonar.version>
```

Comment:
It is backward compatible. Except corner cases, a plugin developed for a given LTS API (and not using deprecated stuff) will work until LTS+2 (included).